eradiate/radprops/rad_profile.py

```diff
@@ -494,13 +494,15 @@ class US76ApproxRadProfile(RadProfile):
 
     levels = documented(
         pinttr.ib(
-            factory=lambda: ureg.Quantity(np.linspace(0, 86, 87), "km"),
-            converter=pinttr.converters.to_units(ucc.deferred("length")),
-            validator=pinttr.validators.has_compatible_units,
+            factory=lambda: np.arange(0.0, 86.01, 1.0) * ureg.km,
```

Comment:
Why 86.01 and not 86.0?

Reply:
Because `arange()` will stop at 85 otherwise [https://numpy.org/doc/stable/reference/generated/numpy.arange.html].